source/MissionAction.cpp

```diff
@@ -164,10 +164,18 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 			int maxDays = (child.Size() >= 4 ? child.Value(3) : minDays);
 			if(maxDays < minDays)
 				swap(minDays, maxDays);
-			events[child.Token(1)] = make_pair(minDays, maxDays);
+			string eventName = child.Token(1);
+			events[eventName] = make_pair(minDays, maxDays);
+			// Create a GameData reference to this mission name.
+			GameData::Events().Get(eventName);
```

Comment:
A cleaner option might just be to add a Name() function to GameEvent. Then, that could be used to check for unloaded events *and* the events map here could use GameEvent pointers instead of strings.